client/i18n/locales/english/intro.json

```diff
@@ -5004,6 +5004,10 @@
       "exam-certified-full-stack-developer": {
         "title": "Certified Full Stack Developer Exam",
         "intro": ["Pass this exam to become a Certified Full Stack Developer."]
+      },
+      "workshop-html-video-player": {
```

Comment:
please move this upward so that it is below `lecture-working-with-audio-and-video-elements`